spec/ParseQuery.spec.js

```diff
@@ -3186,6 +3191,7 @@ describe('Parse.Query testing', () => {
         ok(!result.dirty(), 'expected result not to be dirty');
         strictEqual(result.get('foo'), 'baz');
         strictEqual(result.get('bar'), 1);
+        strictEqual(result.get('qux'), undefined, "expected 'qux' field to be unset");
```

Comment:
I added `qux` to make this test more robust and ensure that we are actually selecting specific keys when using an array